android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveydetail/SurveyDetailViewModel.kt

```diff
@@ -2,18 +2,33 @@ package co.nimblehq.ic.kmm.suv.android.ui.screens.surveydetail
 
 import co.nimblehq.ic.kmm.suv.android.ui.base.BaseViewModel
 import co.nimblehq.ic.kmm.suv.android.ui.screens.home.SurveyArgument
+import co.nimblehq.ic.kmm.suv.android.ui.screens.surveyquestions.SurveyQuestionsArgument
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
 class SurveyDetailViewModel : BaseViewModel() {
 
+    private lateinit var surveyArgument: SurveyArgument
     private val _contentUiModel = MutableStateFlow<SurveyDetailContentUiModel?>(null)
 
     val contentUiModel: StateFlow<SurveyDetailContentUiModel?>
         get() = _contentUiModel
 
+    val surveyQuestionsArgument: SurveyQuestionsArgument?
+        get() {
+            return if (this::surveyArgument.isInitialized) {
+                SurveyQuestionsArgument(
+                    id = surveyArgument.id,
+                    coverImageUrl = surveyArgument.coverImageUrl
+                )
+            } else {
+                null
+            }
+        }
```

Comment:
I think preparing an argument for questions in VM is unnecessary. Another approach is defining `surveyArgument: SurveyArgument` as nullable and getting rid of `lateinit` since we return the nullable argument.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/115/commits/7c8bfad65dc458d366673341187514c84de8403f